scripts/launch_blocked_analysis.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
remove `launch_analysis` and rename this one into `launch_analysis` ?

Reply:
I think we should keep the current naming, it also contribute to explain _how_ the analysis is done.